server/plugin/api.go

```diff
@@ -432,7 +452,6 @@ func (p *Plugin) handleGetSubscriptions(w http.ResponseWriter, r *http.Request)
 	channelID := r.URL.Query().Get(constants.QueryParamChannelID)
 	serviceType := r.URL.Query().Get(constants.QueryParamServiceType)
 	eventType := r.URL.Query().Get(constants.QueryParamEventType)
-	project := r.URL.Query().Get(constants.QueryParamProject)
 	if project != "" {
```

Comment:
What's the intended functionality when `project ` is an empty string? The code reads a bit weird because there is a large block for handling when it's not empty, but then the function ends after that block

Reply:
The code will reach here only when the `project` is not empty and it's valid i.e. linked/exists on the plugin
and in that case, we are returning the array of subscriptions

Removed this check as it's no longer needed now as we have made it required now